src/styles/_themes.scss

```diff
@@ -73,25 +68,210 @@ $themes: (
     colorFlyoutBackground: $colorNoir,
     colorFlyoutText: $colorSmoke,
     colorFlyoutSectionBorder: $colorLead,
+    colorFlyoutDropShadow: rgba(0, 0, 0, 0.6),
+
     colorPropertyGridBorder: $colorLead,
     colorPropertyGridBottomBorder: $colorCharcoal,
+    colorScrollBarBackground: $colorMidnight,
+    colorScrollBarButton: $colorLead,
+    // Page Content Colors - END
+
+    // Header Colors - START
+    colorHeaderDropdown: $colorNoir,
+    colorHeaderText: $colorSmoke,
+    colorHeaderIconHover: $colorWhite,
+    colorHeaderItemBgHover: $colorMidnight,
+    colorHeaderBorderColor: $colorLead,
+    // Header Colors - END
+
+    // Nav Colors - START
+    colorNavBg: $colorCharcoal,
+    colorNavText: $colorWhite,
+    colorNavItemBgHover: $colorMidnight,
+    colorNavItemIcon: $colorSmoke,
+    // Nav Colors - END
+
+    // Context Menu Colors - START
+    colorContextMenuBtnBgHover: $colorMidnight,
+    colorContextMenuBtnTextHover: $colorWhite,
+    colorContextMenuBorderColor: $colorLead,
+    // Context Menu Colors - END
+
+    // Dashboard Panel Colors - START
+    colorDashboardPanelText: $colorWhite,
+    colorDashboardPanelHeaderBg: $colorCharcoal,
+    colorDashboardPanelContentBg: $colorMidnight,
+    colorDashboardPanelOverlayBg: rgba(0, 0, 0, .5),
+    colorDashboardPanelErrorOverlayBg: $colorMidnight,
+    // Dashboard Panel Colors - END
+
+    // Toggle Btn Colors - START
+    colorEnableToggleBackground: $colorGunmetal,
+    colorToggle: $colorWhite,
+    colorDisableToggleBackground: $colorNoir,
+    colorDisabledLine: $colorSmoke,
+    // Toggle Btn Colors - END
+
+    // Forms - START
+    colorFileUploadText: $colorNoir,
+    colorSelectBackground: $colorNoir,
+    colorSelectOptionFocusedBg: $colorCharcoal,
+    colorSelectDisabledText: $colorAsh,
+    colorSelectBorderColor: $colorStone,
+    colorInputText: $colorSmoke,
+    colorInputPlaceholderText: $colorStone,
+    colorInputBorderColor: $colorGraphite,
+    colorInputBorderHoverColor: $colorSmoke,
+    colorDisabledInputBg: $colorGraphite,
+    colorFormActionsBorderColor: $colorGraphite,
+    colorDurationLabelText: $colorSmoke,
+    // Forms - END
+  ),
+  light: (
+    // TODO - Remove START
+    // Highs
+    colorHighest: $colorWhite,
+    // TODO - Remove END
+
+    // Functional colors
+    colorAlarm: $colorAlarm,
+    colorWarning: $colorWarning,
+    colorSystem: $colorSystem,
+    colorError: $colorError,
+
+    // App - START
+    colorSelectionText: $colorWhite,
+    colorSelectionBg: $colorSystem,
+    // App - END
+
+    // Grid - START
+    colorGridScrollBar: $colorCharcoal,
+    colorGridRowBackground: $colorWhite,
+    colorGridCellText: #212121,
+    colorGridRowHoverBackground: #dadada,
+    colorGridSelectedRowBorder: $colorSystem,
+    colorGridHeaderText: #767676,
+
+    colorCellRendererSvgFill: #212121,
+    colorCellRendererSvgFillHighlight: #212121,
+    colorCellRendererText: #212121,
+    colorCellRendererTextHighlight: #212121,
```

Comment:
The light theme has values as hard coded compared to dark theme that has variables. Is there a TODO to change later?

Reply:
There is a TODO now